utils/error.go

```diff
@@ -67,3 +67,7 @@ func ErrReadingRemoteFile(err error) error {
 func ErrReadingLocalFile(err error) error {
 	return errors.New(ErrReadingLocalFileCode, errors.Alert, []string{"error reading local file"}, []string{err.Error()}, []string{"File doesnt exist in the location", "File name is incorrect"}, []string{"Make sure to input the right file name and location"})
 }
+
+func ErrGettingLatestReleaseTag(err error) error {
+	return errors.New(ErrGettingLatestReleaseTagCode, errors.Alert, []string{"Could not fetch latest stable release from github"}, []string{err.Error()}, []string{"Failed to make GET request to github", "Invalid response recieved on github.com/<org>/<repo>/releases/stable"}, []string{"Make sure Github is reachable", "Make sure a valid response is available on github.com/<org>/<repo>/releases/stable"})
```

Comment:
The use of "stable" here seems inaccurate. Will you have another think on this?

Reply:
Yess, that was a mistake